js/components/skipControls.js

```diff
@@ -98,25 +98,27 @@ class SkipControls extends React.Component {
   }
 
   /**
-   * Determines whether or not the current video is at the live edge based on the
-   * playhead state and duration.
+   * Determines whether or not the current video is at the end (VOD) or at the
+   * live edge (DVR Live Streams) based on the playhead state and duration.
    * @private
-   * @return {Boolean} True if the video is at the live edge, false otherwise.
-   * Note: This function always returns false for VOD.
+   * @return {Boolean} True if the video is at the video end/live edge, false otherwise.
    */
-  isAtLiveEdge() {
+  isAtVideoEnd() {
```

Comment:
I guess `End` is less correct than `Edge`. For live stream we're not in the end, so it's a kind of confusing for newcomer what this function does. My suggestion: `isAtVideoEdge`.

Reply:
Will update the name